token-metadata/program/src/state.rs

```diff
@@ -0,0 +1,38 @@
+use {
+    borsh::{BorshDeserialize, BorshSerialize},
+    solana_program::pubkey::Pubkey,
+};
+/// prefix used for PDAs to avoid certain collision attacks
+pub const PREFIX: &str = "metadata";
+
+pub const NAME_LENGTH: usize = 32;
+
+pub const SYMBOL_LENGTH: usize = 10;
+
+pub const URI_LENGTH: usize = 200;
+
+pub const METADATA_LEN: usize = 32 + NAME_LENGTH + SYMBOL_LENGTH + URI_LENGTH + 200;
```

Comment:
+200 feels wrong here, where did that come from?

Reply:
It's an arbitrary amount of padding I picked because I figured we may add things later. I can make it any amount that you feel is reasonable! :)

Reply:
0!  Don't waste one bit

Reply:
Wouldn't this prevent us adding future fields, since we can't resize accounts right now?

Reply:
We need to implement account resizing anyway :)

But I don't think it makes sense to try to future proof an ABI like this in general.  We might never need it and users end up paying inflated costs forever

Reply:
Alright